commit 50549e998d4f5d76151a41d6929b89c58fdd0879 (refs/changes/25/825/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-10-31T16:48:21-07:00 (2 years, 11 months ago)
    
    TOOLS-1592 triton CLI *using node 6* fails various commands with MD5 errors

diff --git a/lib/SaferJsonClient.js b/lib/SaferJsonClient.js
index 64ab172..48da580 100644
--- a/lib/SaferJsonClient.js
+++ b/lib/SaferJsonClient.js
@@ -204,7 +204,7 @@ SaferJsonClient.prototype.parse = function parse(req, callback) {
 
         res.on('data', function onData(chunk) {
             if (contentMd5Hash) {
-                contentMd5Hash.update(chunk.toString('utf8'));
+                contentMd5Hash.update(chunk.toString('utf8'), 'binary');
             }
 
             if (gz) {
